open-apparel-registry/open-apparel-registry
# Allow blank new_oar_ids

## Overview

A new OAR ID, linking a facility to a new location / ID, should not be
required, because most facilities have not been moved. However, the
Django admin was requiring users to enter a new ID in order to save
changes to facilities. A migration has been added to update the model
to allow blank new_OAR_IDs.

Connects #1484 

## Testing Instructions

* On `develop`, run `./scripts/server`. Attempt to update a facility with a blank new_OAR_ID in the Django admin and confirm that it fails.
* On this branch, run `./scripts/manage migrate` and then `./scripts/server`. Attempt to update a facility with a blank new_OAR_ID in the Django admin and confirm that it succeeds.

## Checklist
